idl/paging_state.idl.hh

```diff
@@ -18,6 +31,8 @@ class paging_state {
     uint32_t get_rows_fetched_for_last_partition_low_bits() [[version 3.1]] = 0;
     uint32_t get_remaining_high_bits() [[version 4.3]] = 0;
     uint32_t get_rows_fetched_for_last_partition_high_bits() [[version 4.3]] = 0;
+    bound_weight get_clustering_key_weight() [[version 5.1]] = bound_weight::equal;
+    partition_region get_partition_region() [[version 5.1]] = partition_region::partition_start;
```

Comment:
Isn't 'clustered' more consistent with the current state as a default? So if we read paging_state from an old coordinator we get the old behavior.

Reply:
btw, this will be tricky to test, I guess we'll need a dtest with 1MB rows to have good control over page breaks.

Reply:
Yes, `clustered` should be a better default. The dtest suite has good paging tests, I remember when making reads stateful, they caught a lot of bugs.